examples/Cargo.toml

```diff
@@ -7,7 +7,8 @@ publish = false
 
 [dependencies]
 anyhow = "1.0.62"
-identity_iota = { path = "../identity_iota", features = ["memstore", "domain-linkage", "stronghold"] }
+identity_eddsa_verifier = { path = "../identity_eddsa_verifier" }
```

Comment:
Why don't we include it in `identity_iota`?

Reply:
The idea is to be fully agnostic over the verifier and also the storage (which is why stronghold was removed).
Storage and Verifier are things users have to select for themselves so while we offer some defaults, we don't necessarily have to ship them as part of the identity_iota crate.
The whole topic is very debatable, and we can reasonably go in either direction.